carta/image.py

```diff
@@ -593,6 +593,83 @@ def hide_contours(self):
         """Hide the contours."""
         self.set_contours_visible(False)
 
+    # VECTOR OVERLAY
+
+    @validate(Constant(VectorOverlaySource), Constant(VectorOverlaySource), NoneOr(Number()), Boolean(), NoneOr(Number()), NoneOr(Number()), NoneOr(Number()))
+    def configure_vector_overlay(self, angular_source=VectorOverlaySource.CURRENT, intensity_source=VectorOverlaySource.CURRENT, pixel_averaging=4, fractional_intensity=False, threshold=None, q_error=None, u_error=None):
+        """Configure vector overlay.
+
+        Parameters
+        ----------
+        angular_source : {0}
+            The angular source. By default the current image is used.
+        intensity_source : {1}
+            The intensity source. By default the current image is used.
+        pixel_averaging : {2}
```

Comment:
"in pixel" -> "in pixels"

Reply:
(And it needs the default -- I think it's fine to put "The default is 4." as the middle sentence; you don't need to restate the units.)

Reply:
Sorry; I selected the wrong lines somehow -- this is related to the pixel averaging description.